<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifierDiaporama</class>
 <widget class="QDialog" name="ModifierDiaporama">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>502</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>502</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="Titre">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Modifier Diaporama</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="TextePartieDiapo">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Partie Diaporama</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="TexteIntituleDiapo">
     <property name="text">
      <string>Intitule Diaporama</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ChampIntituleDiapo"/>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="TextePartieImage">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Partie Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="TexteIntituleImage">
     <property name="text">
      <string>Intitul√© Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ChampIntituleImage"/>
   </item>
   <item>
    <widget class="QLabel" name="TexteChemin">
     <property name="text">
      <string>Chemin</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ChampChemin"/>
   </item>
   <item>
    <widget class="QLabel" name="TexteRang">
     <property name="text">
      <string>Rang</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ChampRang"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModifierDiaporama</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModifierDiaporama</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
